packages/ui/packages/CheckBox/test/index.spec.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { Wrapper, theme } from '../../../test/test.helpers';
+import CheckBox from '../index';
+
+
+describe('CheckBox ui component tests', () => {
+  it('Should render with green status and size of 2rem at default', () => {
+    const { getByTestId } = render(
+      <Wrapper><CheckBox /></Wrapper>,
+    );
+
+    expect(getByTestId('checkBox')).toHaveStyle({
+      backgroundColor: 'transparent',
+      width: 'calc(2 * 1rem)',
+      height: 'calc(2 * 1rem)',
+      border: 'solid calc(0.1 * 1rem) theme.colors.gray.light',
+    });
+  });
+
+  // it('Should add CheckMarkIcon in checkBox when get selected props', () => {
+  //   const { getByTestId } = render(
+  //     <Wrapper>
+  //       <CheckBox selected />
+  //     </Wrapper>,
+  //   );
+  //   const checkMarkIcon = getByTestId('checkMarkIcon');
+
+  //   expect(getByTestId('checkBox')).toContainElement(checkMarkIcon);
+  // });
```

Comment:
this test will not run, because our icons do not have testid. I will add all icons an `data-testid` tonight